### PR TITLE
Add .editorconfig & .gitattributes files

### PR DESCRIPTION
**Pull request by JesusValera**
### 📚 Description

This PR adds two config files to the `phel` project which I consider they are very important.

- What is [editorconfig](https://editorconfig.org/)?

**EditorConfig** helps maintain consistent coding styles for multiple developers working on the same project across various editors and IDEs. The EditorConfig project consists of a **file format** for defining coding styles. EditorConfig files are easily readable and they work nicely with version control systems.

- What are [gitattributes](https://git-scm.com/book/en/v2/Customizing-Git-Git-Attributes)?

You can tell Git not to export certain files or directories when generating an archive. If there is a subdirectory or file that you don’t want to include in your archive file but that you do want checked into your project, you can determine those files via the export-ignore attribute.

In other words, basically, when you require from `composer` the `phel` dependency, you won't download the folders that are ignored.

You can see an example of `PHPUnit` dependency in any project:

image

If you take a look at the `phpunit/phpunit`, you can realise there are more files and they are ignored because of this file 😄 

image

### CODE REVIEW

**Review by Chemaclass — APPROVED** [review]
Great idea
